.github/workflows/test.yaml

```diff
@@ -20,32 +20,50 @@ jobs:
   testing:
     runs-on: ubuntu-20.04
     env:
-      TOXENV: "${{ matrix.tox-env }}-withcov-posix,coverage-prepare,codecov-push,coveralls-push"
-    name: ${{ matrix.python-version }}${{ matrix.noipv6 }}-${{ matrix.tox-env }}-withcov-posix
+      TOXENV: "${{ matrix.tox-env }}"
+      CODECOV_OPTIONS: "-n 'lnx-${{ matrix.python-version }}-${{ matrix.tox-env }}${{ matrix.noipv6 }}'"
+      TRIAL_ARGS: "${{ matrix.trial-args }}"
```

Comment:
I would call this TRIAL_JOBS and in the Tox config use `--jobs="{env:TRIAL_JOBS}"` when invoking Trial.  But I'd also set TRIAL_JOBS in a `setenv` clause in Tox, and have it default to `2`, so that when one runs locally, they are also testing in parallel (which is faster but also matches CI).

Reply:
From PR description :)

> I added TRIAL_ARGS and not just a more specific TRIAL_JOBS as even when using 1 job trial still spawn a separate process and debugging is not enabled.

My experience using tox for local TDD is horrible... so I stopped using tox at all on my local system.

Do you use tox when developing twisted on your local system? Isn't it super slow?

Reply:
OK, sorry I missed that text.

I use tox locally. And yes, it is way more painful with Twisted than with other projects, and in my experience actually wrong if you don't always use `-r`, which is insane.

All that said, making tests when run locally match what's happening in CI is probably a good idea, and "Tox sucks for me" seems like a non-argument, and "it's super slow" suggests that defaulting to `-j2` would be an improvement.

I don't know what "debugging is not enabled" means; I probably just never use the enabled debugging things.

All *that* said, I'm fine with leaving `TRIAL_ARGS ` alone here if "debugging is not enabled" causes people grief.